src/routes/transactions.ts

```diff
@@ -343,3 +343,13 @@ export const creteBookingFromReservation = (person, reservation, incoming?) => {
     meta_info: metaInfo,
   };
 };
+
+export const directDebitRefund = (req, res) => {
+  log.info("refund confirmed", {
+    personId: req.params.person_id,
+    accountId: req.params.account_id,
+    bookingId: req.body.booking_id,
+  });
+
+  res.status(200).send("OK");
```

Comment:
I would either send 204 or empty JSON object. Returning plain string to a caller who expects JSON might result in error depending on response parsing implementation

Reply:
thanks, I return 201 as in the doc.